### PR TITLE
Add package plugin instructions to documentation

### PR DESCRIPTION
**Pull request by franklinsch**
Bug/issue #, if applicable: N/A

## Summary

Add instructions to set up the plugin in the documentation. When reading the documentation at https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/, it might not be totally clear that the plugin needs to be added to your package first, especially if the developer is not yet familiar with the concept of SwiftPM plugins.

## Dependencies

None.

## Testing

No functional changes.

## Checklist

- ~[ ] Added tests~ N/A
- [x] Ran the `./bin/test` script and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

**Comment by franklinsch** [comment]
@swift-ci please test

File: Sources/SwiftDocCPluginDocumentation/SwiftDocCPlugin.docc/SwiftDocCPlugin.md [hunk 1]
```diff
@@ -7,8 +7,19 @@ Produce Swift-DocC documentation for Swift Package libraries and executables.
 The Swift-DocC plugin is a Swift Package Manager command plugin that supports building
 documentation for SwiftPM libraries and executables.
 
-After adding the plugin as a dependency in your Swift package manifest, you can build
-documentation for the libraries and executables in that package and its dependencies by running the
+First, add the plugin as a dependency in your Swift package manifest:
```
Comment (ethan-kusters) [inline Sources/SwiftDocCPluginDocumentation/SwiftDocCPlugin.docc/SwiftDocCPlugin.md:10]: This is great- thanks! Should we make this match the format of the package manifest in the Getting Started guide?

https://github.com/apple/swift-docc-plugin/blob/main/Sources/SwiftDocCPluginDocumentation/SwiftDocCPlugin.docc/Getting%20Started%20with%20the%20Swift-DocC%20Plugin.md

I'm happy with either format really but it seems like it makes sense to align between the two.

Comment (franklinsch) [reply Sources/SwiftDocCPluginDocumentation/SwiftDocCPlugin.docc/SwiftDocCPlugin.md:10]: Do we still need a separate Getting Started guide? I feel like it is concise enough to be included on the main page and the info it contains is likely the most important info on the website.

Comment (ethan-kusters) [reply Sources/SwiftDocCPluginDocumentation/SwiftDocCPlugin.docc/SwiftDocCPlugin.md:10]: Hm. That's fair. I guess this PR is more of a question of: "Do we want the plugin overview page to be a getting started guide?"

If we _do_ add the Package.swift instructions here then I agree the Getting Started guide is entirely redundant and we should remove it as part of this PR.

I'm not sure what's better from a documentation perspective though- having a specific Getting Started guide or just building that into the docs homepage.

Definitely an argument to be made that the Plugin is small enough in scope that we should just merge the two.

Comment (franklinsch) [reply Sources/SwiftDocCPluginDocumentation/SwiftDocCPlugin.docc/SwiftDocCPlugin.md:10]: I think that this website's main audience is developers asking the question "how do I set up the plugin in my package?" to which I think the homepage should provide an answer for. The other articles provide additional guidance in more specific use cases.

Comment (ethan-kusters) [reply Sources/SwiftDocCPluginDocumentation/SwiftDocCPlugin.docc/SwiftDocCPlugin.md:10]: Yeah I agree with that. Do you want to go ahead and remove the "Getting Started" guide as part of this PR?

I think we should still align on formatting the code example here with the README though: https://github.com/apple/swift-docc-plugin/blob/main/README.md#adding-the-swift-docc-plugin-as-a-dependency.

Comment (franklinsch) [reply Sources/SwiftDocCPluginDocumentation/SwiftDocCPlugin.docc/SwiftDocCPlugin.md]: Yep sounds good. I pushed the change.

**Review by ethan-kusters — APPROVED** [review]
Thank you!

**Comment by ethan-kusters** [comment]
@swift-ci please test

**Pull request merged**